Upgrading-python-interpreter-to-version-3.8-for-existing-oppia-directory.md

```diff
@@ -0,0 +1,94 @@
+# Upgrading python interpreter version from 3.7.10 to 3.8.12
+
+Oppia has updated it's python interpreter version 3.7.10 in a phase of switching to a more updated interpreter version 3.8.12. For effective and effortless switching to the newer version, this guide will provide a brief and detailed instruction on how to achieve just that. Please follow the below mentioned instructions carefully.
+
+## Prerequisite
+- This page is dedicated to people who want to upgrade their python interpreter in the existing Oppia directory instead of maintaing a new local repo copy of oppia.
```

Comment:
This page is aimed at developers who want to upgrade their Python interpreter in their existing Oppia directory, instead of creating a separate copy of Oppia with the new interpreter.

Before starting, please make sure that your local repo, as well as your remote fork on GitHub, are both up-to-date with the `develop` branch.

(Note: You don't need the bullet points. Just use two paragraphs.)

Reply:
Done !!